README.md

```diff
@@ -468,7 +468,7 @@ Options:
       --show-settings
           See the settings Ruff will use to lint a given Python file
   -h, --help
-          Print help
+          Print help (see more with '--help')
```

Comment:
I assume this is related to `hide_possible_values`, but I'm trying to wrap my head around what it's saying, when it would show up, and what would change by adding `--help` -- isn't this message printed when the user _runs_ `--help`?

Reply:
Oh yeah ... this also confused me quite a bit. As it turns out this is a Clap bug. I submitted a fix to upstream with https://github.com/clap-rs/clap/pull/4710.

Reply:
Thanks for doing that!